Source/EasyNetQ.Management.Client/Serialization/DateTimeConverter.cs

```diff
@@ -0,0 +1,19 @@
+using System.Diagnostics;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace EasyNetQ.Management.Client.Serialization;
+
+internal class DateTimeConverter : JsonConverter<DateTime>
+{
+    public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        Debug.Assert(typeToConvert == typeof(DateTime));
```

Comment:
Do we really need this here?

Reply:
No, if we make `ShovelStatus.Timestamp` `string` rather than `DateTime`.